docs/src/development/email.md

```diff
@@ -208,3 +212,11 @@ Guides on using JavaMail:
 
 - [Send email with HTML formatting and attachments](https://mkyong.com/java/java-how-to-send-email/)
 - [JavaMail API](https://javaee.github.io/javamail/)
+
+{{< /codetabs >}}
+
+## Alternative: Set up your own email server
```

Comment:
:hammer_and_wrench: :arrow_forward:
Since that section isn't only about the "own" mail server, maybe let's make it more generic?
```suggestion
## Alternative: Set up an external email server
```

Reply:
That's a fair point, but I don't think the server has to be external, either. They can run one on the infrastructure, as far as I know, just on a different port. I could try `Use a different email server`, although it's less clear, it is more comprehensive